Examples/LocationManager/LocationManager/LocationManagerClient/LocationManagerClient.swift

```diff
@@ -0,0 +1,23 @@
+import Combine
+import ComposableArchitecture
+import CoreLocation
+
+struct LocationManagerClient {
```

Comment:
this is a TCA-friendly wrapper around the `CLLocationManager` API. there's a lot more functionality that could be exposed, but this is all we needed for this demo.